andela/activo-web
# #159079011 Delete an additional attribute

**What does this PR do?**

User should be able to delete an additional attribute

**Description of Task to be completed?**

- Click on the delete button on an additional attribute, a confirmation modal pops up.
- Click on the delete button on the modal to delete.
- Write tests for createCategory page
- Write tests for attribute component

**How should this be manually tested?**

 - `yarn start:dev` to open the web browser on `port 3000` 
-  On the `settings page` when creating `a new category` click on add new attribute icon for an attribute field to be added then click on the trash icon to display a confirmation modal, confirm to delete.

**Any background context you want to provide?**

xxx

**What are the relevant pivotal tracker stories?**

[#159079011](https://www.pivotaltracker.com/n/projects/2170023/stories/159079011)

**Screenshots (if appropriate)**

<img width="1440" alt="screen shot 2018-08-03 at 9 49 17 am" src="https://user-images.githubusercontent.com/28443813/43636776-91425eea-9702-11e8-96e5-547988979750.png">


**Questions:**

xxx
